### PR TITLE
add push token to status

### PR DESCRIPTION
**Pull request by joshblum**
example log send id `0a775d4db19334a37e3c391c`

### CODE REVIEW

File: shared/constants/settings.js [hunk 1]
```diff
@@ -75,6 +75,12 @@ export const makeState: I.RecordFactory<Types._State> = I.Record({
   waitingForResponse: false,
 })
 
+export const getPushTokenForLogSend = (state: Object) => {
+  return I.Map({
```
Comment (chrisnojima) [inline shared/constants/settings.js]: why make a map and convert it to an object?

Comment (joshblum) [reply shared/constants/settings.js]: i was following the existing call below: https://github.com/keybase/client/pull/17591/files#diff-1d9ec48879ceb528287d545487be4738R89

Comment (chrisnojima) [reply shared/constants/settings.js]: thats cause it had hierarchical immutable structures, which this does not. can be
```
=> ({pushToken: state.push.token})
```

Comment (joshblum) [reply shared/constants/settings.js]: got it, thanks!

**Comment by joshblum** [comment]
@chrisnojima this is ready

**Pull request merged**